ProcessLib/Process.h

```diff
@@ -32,6 +32,14 @@ class Process
     virtual void initialize() = 0;
     virtual bool solve(const double delta_t) = 0;
 
+    /// Explicitly release memory of global vector, matrix and linear solvers
```

Comment:
Because I already added this function in PR #580, I suggest not to change this file but  to remove the following lines in ogs.cpp
```
for (auto p_it = project.processesBegin(); p_it != project.processesEnd(); ++p_it)
	{
		(*p_it)->releaseEquationMemory();
	}
```